content/install_maintain/installation/installing-cluster.md

```diff
@@ -144,19 +147,26 @@ The PostgreSQL database high-availability cluster is comprised of 3 nodes (Cloud
  tcp/2380  | etcd port.
  tcp/5432  | PostgreSQL connection port.
  tcp/8008  | Patroni control port.
+ tcp/8009  | Monitoring service port.
 
 
 #### Externally Hosted PostgreSQL Database Installation
  - Make sure the PostgreSQL instance is publicly available and reachable from the local Cloudify Management service cluster nodes.
  - Retrieve the PostgreSQL instance CA certificate and save it locally for future use in the 
  Cloudify Management service cluster nodes configuration.
- - Keep your PostgreSQL database username and password for the later configuration of the Cloudify Management service cluster nodes. 
- 
+ - Keep your PostgreSQL database username and password for the later configuration of the Cloudify Management service cluster nodes.
+ - Have [Prometheus](https://prometheus.io/), [node_exporter](https://github.com/prometheus/node_exporter/) and
+ [postgres_exporter](https://github.com/wrouesnel/postgres_exporter) metrics available via Prometheus API at
+ `https://{private_ip}:8009/monitoring/api`; Cloudify Manager will be using a CA certificate mentioned above to verify the TLS
+ connection and credentials defined in the Manager's configuration under a path `postgresql_client.monitoring.{username,password}`.
+
 ##### Azure DBaaS for Postgres
 
-Cloudify supports [Microsoft's Azure Database for Postgres](https://docs.microsoft.com/en-us/azure/postgresql/) as an external database option replacing Cloudify's PostgreSQL deployment.  
+Cloudify supports [Microsoft's Azure Database for Postgres](https://docs.microsoft.com/en-us/azure/postgresql/) as an external database option replacing Cloudify's PostgreSQL deployment.
 
-Azure Database for Postgres is a fully managed database-as-a-service offering that can handle mission-critical workloads with predictable performance, security, high availability, and dynamic scalability. It is available in two deployment options, as a single server and as a Hyperscale (Citus) cluster (preview).  
+Azure Database for Postgres is a fully managed database-as-a-service offering that can handle mission-critical workloads with predictable performance, security, high availability, and dynamic scalability. It is available in two deployment options, as a single server and as a Hyperscale (Citus) cluster (preview).
+
+**Note**  Setting up a Prometheus with required exporters is not trivial in such case.  Without them you will not be able to monitor database nodes with Cloudify's standard tooling.
```

Comment:
I think this is not relevant. we do not use Prometheus or anything else to monitor Azure DBaaS or AWS RDS, or customer set DB.
We just monitor the access to the DB from the manager side.